dist/rpm/os-autoinst-scripts-deps.spec

```diff
@@ -24,13 +24,15 @@ License:        MIT
 Group:          Development/Tools/Other
 BuildArch:      noarch
 Url:            https://github.com/os-autoinst/scripts
+Source0:        scripts-%{version}.tar.xz
```

Comment:
where this come from? I didnt see it in https://build.opensuse.org/package/show/devel:openQA/os-autoinst-scripts which I would expect to find it, or in https://build.opensuse.org/package/show/devel:openQA:GitHub:os-autoinst:scripts:PR-437/os-autoinst-scripts(from your test PR)

Reply:
That's created by the obs "tar" plugin
https://build.opensuse.org/projects/devel:openQA:GitHub:os-autoinst:scripts:PR-438/packages/os-autoinst-scripts/files/_service?expand=1